WeHere/Views&ViewModels/Components/Modal/BottomModal/BottomModal.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  BottomModal.swift
+//  WeHere
+//
+//  Created by Chanhee Jeong on 2022/11/21.
+//
+
+import SwiftUI
+
+struct BottomModal: View {
+    @State private var currentIndex = 0
+    let place = Place.mockData[1]
+    
+    var body: some View {
+        TabView(selection: $currentIndex) {
+            ForEach(0..<2) { index in
+                ZStack {
+                    Color.white
+                    switch index {
+                    case 0:
+                        pokeView
+                            .padding(EdgeInsets(top: 28, leading: 28, bottom: 28, trailing: 28))
+                        
+                    case 1:
+                        placeDetailView
+                            .padding(EdgeInsets(top: 28, leading: 28, bottom: 28, trailing: 28))
+                        
+                    default:
+                        EmptyView()
+                    }
+                }
+                .clipShape(RoundedRectangle(cornerRadius: 30.0,
+                                            style: .continuous))
+            }
+            .padding(.all, 10)
```

Comment:
```suggestion
            .padding(10)
```
[P4] 이 부분에서 방향을 지정하신 이유가 있으실까요??